lib/capybara/node/matchers.rb

```diff
@@ -364,10 +364,10 @@ def has_field?(locator, options={})
       # Checks if the page or current node has no form field with the given
       # label, name or id. See {Capybara::Node::Matchers#has_field?}.
       #
-      # @param [String] locator           The label, name or id of a field to check for
-      # @option options [String] :with    The text content of the field
-      # @option options [String] :type    The type attribute of the field
-      # @return [Boolean]                 Whether it doesn't exist
+      # @param [String] locator                  The label, name or id of a field to check for
+      # @option options [String/Regexp] :with    The text content of the field or a Regexp to match
```

Comment:
In Yard a types specifier list is a comma separated list of types, not / separated